### DISCUSSION TITLE
Can xiSearch identify a mixture of 14N and 15N labelled xlinked peptides

### DISCUSSION THREAD

**Discussion by BioBern** [Rappsilber-Laboratory/XiSearch#95] [General]
Can xiSearch handle the analysis of isotope labelled proteins? We mix normal protein with the 15N isotopic labelled version of the same protein. We get a mixture of light 14N peptides crosslinked with heavy 15N labelled peptides.  I have both heavy 15N and light 14N peptides at the same time and L-L, L-H and H-H crosslinked peptides are present in the sample. Can I use xiSearch to detect these heavy and light xlinked peptides?

**Comment by lutzfischer** [comment]
Hi,


First off - I have no experience with N15 labelled data so take everything with a grain of salt. 

You should always be able to identify the N14 versions without anything special. For the N15 labelled peptides the answer somewhat depends on the level of N15 incorporation. If you have a very high incorporation (e.g. close to 100%) you could  define a label schema and it should work.


But label are currently not exposed directly in the GUI - so you would need to add some custom setting for each amino acid. 
E.g. copy paste the following lines into the custom settings field (or if you work with a text config just past it at the beginning of the config).

```
LABEL:HEAVY::SYMBOL:An15;MODIFIED:A;MASS:72.034148775
LABEL:HEAVY::SYMBOL:Cn15;MODIFIED:C;MASS:104.006219475
LABEL:HEAVY::SYMBOL:Dn15;MODIFIED:D;MASS:116.023978035
LABEL:HEAVY::SYMBOL:En15;MODIFIED:E;MASS:130.039628105
LABEL:HEAVY::SYMBOL:Fn15;MODIFIED:F;MASS:148.065448915
LABEL:HEAVY::SYMBOL:Gn15;MODIFIED:G;MASS:58.018498705
LABEL:HEAVY::SYMBOL:Hn15;MODIFIED:H;MASS:140.050016785
LABEL:HEAVY::SYMBOL:In15;MODIFIED:I;MASS:114.081098985
LABEL:HEAVY::SYMBOL:Kn15;MODIFIED:K;MASS:130.08903299
LABEL:HEAVY::SYMBOL:Ln15;MODIFIED:L;MASS:114.081098985
LABEL:HEAVY::SYMBOL:Mn15;MODIFIED:M;MASS:132.037519615
LABEL:HEAVY::SYMBOL:Nn15;MODIFIED:N;MASS:116.03699741
LABEL:HEAVY::SYMBOL:Pn15;MODIFIED:P;MASS:98.049798845
LABEL:HEAVY::SYMBOL:Qn15;MODIFIED:Q;MASS:130.05264748
LABEL:HEAVY::SYMBOL:Rn15;MODIFIED:R;MASS:160.08925093
LABEL:HEAVY::SYMBOL:Sn15;MODIFIED:S;MASS:88.029063405
LABEL:HEAVY::SYMBOL:Tn15;MODIFIED:T;MASS:102.044713475
LABEL:HEAVY::SYMBOL:Vn15;MODIFIED:V;MASS:100.065448915
LABEL:HEAVY::SYMBOL:Wn15;MODIFIED:W;MASS:188.07338292
LABEL:HEAVY::SYMBOL:Yn15;MODIFIED:Y;MASS:164.060363545
LABEL:HEAVY::SYMBOL:Un15;MODIFIED:U;MASS:151.950668375
LABEL:HEAVY::SYMBOL:On15;MODIFIED:O;MASS:240.138831835
```

This should define per amino-acid a labelled version that assumes every N is 15N

The effect is, that each protein will be duplicated and searched ones with the original amino-acids and ones with the labeled amino-acids.

This should enable identifications of L-L, L-H and H-H. 


But basically xisearch2 would assume complete incorporation of N15. So the further away you are from completeness, the less likely it will be able to identify any labelled peptides. Basically xiSEARCH depends on the reported masses for precursor and fragments and if the incorporation is not very close to complete the massspec  will not see an 15N peptide  but something slightly lighter and report that "wrong" mass as the peptide m/z or fragment m/z.

**Comment by cxdummies** [comment]
Dear Lutz,

Besides pasting the abovementioned lines to the beginning of the config, does one need to change any further settings? Is any change to the fasta file required?

Thank you.
